### DISCUSSION TITLE
Question about the automatic navigation menu

### DISCUSSION THREAD

**Discussion by lua-rocks** [squidfunk/mkdocs-material#4684] [Q&A] [answered]
I really like that MkDocs can make the navigation menu automatically, it is very convenient. For example: if I name the files "01_red.md", "02_green.md", "03_blue.md" with this content: "# Red", "# Green", "# Blue", the result will be a menu of header colors that is sorted by file names and that's so cool because I don't have to create the "nav" section  in mkdocs.yml at all!

But the problem is that I don't know how to do the same trick with directories. Maybe there is a way? Thanks.

**Comment by sennewood** [comment]
Could you provide an example, please?

**Reply by lua-rocks** [reply]
If you don't have a "nav" section in your mkdocs.yml then such a structure:

```
📂 01_colors
├── 📜 01_red.md
├── 📜 02_green.md
└── 📜 03_blue.md
📂 02_sounds
└── ...
📂 03_smells
└── ...
```

will be displayed in a browser like this:

```
📂 01_colors
├── 📜 Red
├── 📜 Green
└── 📜 Blue
📂 02_sounds
└── ...
📂 03_smells
└── ...
```

The file headers will be read from their contents, but their sort order will be depend on their file names and that's good.

But since directories are not files, they cannot be assigned a `# Header`. In the example above, there is no way for me to hide the sorting numbers in their names.

**Comment by lua-rocks** [comment, marked as answer]
The reason I don't want to use nav section is because I'm too lazy and don't want to manually monitor changes in it every time I add or remove files. It's not convenient.

**Reply by sennewood** [reply]
A popular plugin to modify the nav section is: https://github.com/lukasgeiter/mkdocs-awesome-pages-plugin
I dont't use it, so I don't know if it's useful regarding directories - but maybe it solves your problem.

**Reply by lua-rocks** [reply]
Thank you. I'll check it.

**Reply by lua-rocks** [reply]
@sennewood This plugin is exactly what I need. Thank you so much!
